libraries/Microsoft.Bot.Builder.LanguageGeneration/TemplateErrors.cs

```diff
@@ -10,7 +10,7 @@ public class TemplateErrors
     {
         public const string NoTemplate = "LG file must have at least one template definition.";
 
-        public const string InvalidTemplateName = "Invalid template name. Template name should start with letter/number/_ and can only contains letter/number/./_.";
+        public const string InvalidTemplateName = "Invalid template name. Each part of template name split by dot(.) should start with letter/_ and can only contain letter/_/number.";
```

Comment:
contain\contains

Reply:
Invalid template name. Template names can only contain letter, underscore '_' or a number. Any part of a template name (split by '.') cannot start with a number.